_includes/news-section.html

```diff
@@ -0,0 +1,11 @@
+<article class="c-latest-news">
+<div class="c-latest-news__title-container">
+  <h1 class="c-latest-news__title">Latest News</h1>
+  <a href="#" class="c-latest-news__link c-latest-news__link--border">View All News</a>
+</div>
+<ul class="c-latest-news__link-list">
+  <li class="c-latest-news__list-item"><a href="#" class="c-latest-news__link">Dion Flannery Honored as Dayton Business Journal’s Executive of the Year Finalist</a><p class="c-latest-news__date">february 2017</p></li>
+  <li class="c-latest-news__list-item"><a href="#" class="c-latest-news__link">Scour’s VP of Human Resources Speaks at Women in Aviation</a><p class="c-latest-news__date">june 2017</p></li>
+  <li class="c-latest-news__list-item"><a href="#" class="c-latest-news__link">Scout Named Best Workplace in Ohio for Third Consecutive Year</a><p class="c-latest-news__date">november 2018</p></li>
+  </ul>
```

Comment:
It looks like the nesting may be off on this file, that's a really nit-picky thing but figured I'd point it out!